config/locales/en.yml

```diff
@@ -1025,8 +1027,14 @@ en:
         cannot_update_self: "You cannot update your own %{namespace_type} role"
         role_not_allowed: "A maintainer can only update a member's role up to the Maintainer role"
     samples:
+      clone:
+        empty_new_project_id: Destination project was not selected.
+        empty_sample_ids: The sample ids are empty.
+        same_project: The samples already exist in the project. Please select a different project.
```

Comment:
I think this message should read something like `source and destination projects are the same and to select a different destination project`.

Reply:
66e0574204bd7d111fbbd797045d757985336ce3